### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
Adds important repository documents README.md, CODE_OF_CONDUCT.md, and CONTRIBUTING.md

### CODE REVIEW

**Comment by Ethan-Arrowood** [comment]
please hold - somethings up with my branch

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+4. Install the dependencies with:
+   ```
+   yarn
+   ```
+5. Make changes and run tests using:
+   ```
+   yarn test
+   ```
+
+> This repo is powered by [turborepo](). Running command such as `test`, `build`, and `lint` from the project root will utilize caching techniques to maximize developer productivity.
```
Comment (leerob) [inline CONTRIBUTING.md:34]:
```suggestion
> This repo is powered by [Turborepo](https://turborepo.org/). Running commands such as `test`, `build`, and `lint` from the project root will utilize caching techniques to maximize developer productivity.
```

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
 # Fetch Monorepo
 
-This is the official Vercel fetch monorepo
\ No newline at end of file
+This is the official Vercel fetch monorepo
+
+## Getting Started
+
+There are a variety of ways to use these packages. The core package `@vercel/fetch` bundles the remaining packages together into a super powered [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) client. By default this package will use its peer dependency [node-fetch](https://github.com/node-fetch/node-fetch), but it also supports other fetch implementations.
```
Comment (leerob) [inline readme.md:7]:
```suggestion
`@vercel/fetch` bundles all packages inside this monorepo together into a super-powered [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) client. By default, this package will use its peer dependency [node-fetch](https://github.com/node-fetch/node-fetch), but it also supports other fetch implementations.
```

File: readme.md [hunk 3]
```diff
@@ -1,3 +1,27 @@
 # Fetch Monorepo
 
-This is the official Vercel fetch monorepo
\ No newline at end of file
+This is the official Vercel fetch monorepo
```
Comment (leerob) [inline readme.md:3]:
```suggestion
This fetch monorepo contains three packages:

- `@vercel/fetch`
- `@vercel/fetch-retry`
- `@vercel/fetch-cached-dns`
```

**Review by leerob — COMMENTED** [review]
Let's make sure we add: who is this for? Is it recommended for folks using Next.js? Is it recommended for serverless functions?

File: readme.md [hunk 4]
```diff
@@ -1,3 +1,33 @@
...
 
-This is the official Vercel fetch monorepo
\ No newline at end of file
+This fetch monorepo contains three packages:
+
+- `@vercel/fetch`
+- `@vercel/fetch-retry`
+- `@vercel/fetch-cached-dns`
+
+These packages are recommended to be used in Node.js environments. It aims to bring the familiarity of the Fetch API to the backend. There are future plans to make this project interoperale between both browser and server environments.
```
Comment (styfle) [inline readme.md:9]:
```suggestion
These packages are recommended to be used in Node.js environments. It aims to bring the familiarity of the Fetch API to the backend. There are future plans to make this project interoperable between both browser and server environments.
```

File: readme.md [hunk 5]
```diff
@@ -1,3 +1,33 @@
...
 
-This is the official Vercel fetch monorepo
\ No newline at end of file
+This fetch monorepo contains three packages:
+
+- `@vercel/fetch`
+- `@vercel/fetch-retry`
+- `@vercel/fetch-cached-dns`
+
+These packages are recommended to be used in Node.js environments. It aims to bring the familiarity of the Fetch API to the backend. There are future plans to make this project interoperable between both browser and server environments.
```
Comment (styfle) [inline readme.md:9]:
```suggestion
These packages are designed for use with Node.js in order to bring the familiarity of the Fetch API to the backend. There are future plans to make this project interoperable between both browser and server environments.
```

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**